pydantic/mypy.py

```diff
@@ -438,23 +433,18 @@ def add_construct_method(self, fields: List['PydanticModelField']) -> None:
         obj_type = ctx.api.named_type(f'{BUILTINS_NAME}.object')
         self_tvar_name = '_PydanticBaseModel'  # Make sure it does not conflict with other names in the class
         tvar_fullname = ctx.cls.fullname + '.' + self_tvar_name
-        tvd = TypeVarDef(self_tvar_name, tvar_fullname, -1, [], obj_type)
+        # requires mypy>0.910
```

Comment:
shouldn't it be `mypy>0.930`
```suggestion
        # requires mypy>0.930
```

Reply:
no i think this is right.